portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,27 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fun fact to the page.
  */
 function addRandomGreeting() {
```

Comment:
Consider renaming this function, as it's no longer adding a random greeting.

Reply:
Fixed, thanks!